File: SISTEMA DE TICKET/SISTEMA DE TICKET DIEGO/Persona.java
public abstract class Persona {
    private String strNombre;
    private String strCorreo;
    private String strTelefono;

    public Persona(String strNombre, String strCorreo, String strTelefono) {
        this.strNombre = strNombre;
        this.strCorreo = strCorreo;
        this.strTelefono = strTelefono;
    }

    public String getStrNombre() { return strNombre; }
    public String getStrCorreo() { return strCorreo; }
    public String getStrTelefono() { return strTelefono; }
}

File: SISTEMA DE TICKET/Proyectoticket/Persona.cs
using System;

namespace Proyectoticket.logica
{
    public abstract class Persona(string codigo, string nombre, string email)
    {
        public string StrCodigo { get; set; } = codigo;
        public string StrNombre { get; set; } = nombre;
        public string StrCorreo { get; set; } = email;
        public bool BlnActivo { get; set; } = true;

        public abstract string ObtenerRol();

        public virtual void Registrar()
        {
            Console.WriteLine($"Persona registrada: [{ObtenerRol()}] {StrNombre} ({StrCodigo})");
        }

        public virtual void MostrarInformacion()
        {
            Console.WriteLine($"[{ObtenerRol()}] Codigo: {StrCodigo} | Nombre: {StrNombre} | Correo: {StrCorreo}");
        }
    }
}